irma-css/src/components/irma-form.scss

```diff
@@ -227,4 +160,71 @@
       }
     }
   }
+
+  // Swap out the header content for the helper in a nice animation
+  & > *:not(.irma-web-helper):not(.irma-web-close) {
```

Comment:
Should the block that starts here not be inside the `.irma-web-header` block starting on line 101? Currently the "Show helpers" link in the styleguide seems to be broken, and it is fixed by moving this inside that block. The linter seems to be fine with it either way.

Reply:
Sharp, I didn't notice that. I think I confused `irma-web-header` and `irma-web-helper`. How did you move it then? Because when I move those lines in `.irma-web-header` I get the following result with a very large font.

![image](https://user-images.githubusercontent.com/17035191/121692727-aa0a3e00-cac8-11eb-96a7-3861cdcd771c.png)

Reply:
It does that here too. I'm not sure then what's the best way to fix this.

Reply:
I found a fix that seems to work, and also makes this PR much better reviewable :)

I removed all earlier changes and replaced it by only moving the `irma-web-centered` section.